irohad/torii/impl/command_service_transport_grpc.cpp

```diff
@@ -184,18 +184,27 @@ namespace torii {
     auto client_id_format = boost::format("Peer: '%s', %s");
     std::string client_id =
         (client_id_format % context->peer() % hash.toString()).str();
-
+    bool last_tx_status_received = false;
+    auto status_bus = command_service_->getStatusStream(hash)
+                          .subscribe_on(current_thread)
```

Comment:
```suggestion
```
It might be better to use `.combine_latest(current_thread, consensus_gate_observable)` below instead.